cmd/cdi/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/spf13/cobra v1.6.0
 	gopkg.in/yaml.v2 v2.4.0
 	sigs.k8s.io/yaml v1.3.0
-	tags.cncf.io/container-device-interface v0.0.0
+	tags.cncf.io/container-device-interface v1.0.0
```

Comment:
More a comment on the release process: These are probably not strictly necessary since these CLIs are not generally consumed externally, but it's probably more consistent to have them.

Reply:
I was wondering the same. We anyway have the replace rule, so the version here has no effect, but if we bump it now, we should always go through the motions for consistency. @bart0sh So wouldn't leaving this at `0.0.0` be easier and of a burden for maintenance ?

Reply:
done.